src/DataStructures/DataBox/DataBoxTag.hpp

```diff
@@ -560,6 +560,42 @@ template <class Tag>
 struct remove_all_prefixes_impl<Tag, true> {
   using type = remove_all_prefixes<remove_tag_prefix<Tag>>;
 };
+
+// Implementation of variables_tag_with_tags_list
+template <DispatchTagType TagType>
+struct variables_tag_with_tags_list_impl;
+}  // namespace DataBox_detail
+
+/// \ingroup DataBoxGroup
+/// Change the tags contained in a possibly prefixed Variables tag.
```

Comment:
So if I understand the docs correctly if I have `using tag = Prefix<Variables<Prefix<A>, Prefix<B>>>` and I call `variables_tag_with_tags_list<tag, tmpl::list<C, D>>` I get `Prefix<Variables<Prefix<C>, Prefix<D>>>`? Having that example might be helpful :)

Reply:
No, it just replaces the tags with the passed list without doing anything fancy.

Reply:
I think having an example in the documentation would make it clearer